### ISSUE TITLE
Tooltip shadow dims tooltipped thing

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#2079]
This is truly the smallest nitpick but it seems silly that the number I'm hovering over get dimmed by the shadow on the tooltip.

![2024-03-18-tooltip-shadow](https://github.com/oxidecomputer/console/assets/3612203/5503d09c-9359-45cd-b8eb-9e0a012cfbcd)

I think `elevation-2` must be too spread out for such a small element.

https://github.com/oxidecomputer/console/blob/7596d63320631eb81ae5795b4dd1318d640fda9d/app/ui/styles/components/tooltip.css#L10

**Comment by benjaminleonard** [comment]
Yeah – perhaps we can add size as an extra axis to the elevation tokens.
